src/gui/torrentcreatordialog.cpp

```diff
@@ -164,14 +171,18 @@ void TorrentCreatorDialog::onCreateButtonClicked()
 
     const QStringList trackers = m_ui->trackersList->toPlainText().trimmed()
         .replace(QRegularExpression("\n\n[\n]+"), "\n\n").split('\n');
-    const QStringList urlSeeds = m_ui->URLSeedsList->toPlainText().split('\n', QString::SkipEmptyParts);
-    const QString comment = m_ui->txtComment->toPlainText();
-    const QString source = m_ui->lineEditSource->text();
+    const BitTorrent::TorrentCreatorParams params { m_ui->checkPrivate->isChecked()
```

Comment:
IMO, there is readability issue here. I would either move this argument to the next line or rearrange the other ones to be more aligned (to have lines of more similar length).

Reply:
>move this argument to the next line 

Done.